releasenotes/notes/fix-attribute-error-in-flask-2.2.0-94524c33b005d48a.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 fixes:
   - |
-    Flask: version 2.2.0 removes ``try_trigger_before_first_request_functions``
+    flask: remove patching for ``Flask.try_trigger_before_first_request_functions``. This method was removed in flask==2.2.0.
```

Comment:
don't think we need to mention what changed internally in flask, users don't care.

I think we can leave it as "add support for 2.2.0"